json_annotation/lib/src/json_serializable.g.dart

```diff
@@ -1,26 +1,26 @@
 // GENERATED CODE - DO NOT MODIFY BY HAND
 
-part of 'generator_config.dart';
```

Comment:
This was copy-pasted as part of the PR – generation is not setup (yet). Going to do that later.